src/high-level/src/KinDynComputations.cpp

```diff
@@ -1073,6 +1073,27 @@ Transform KinDynComputations::getWorldTransform(const FrameIndex frameIndex)
     return world_H_frame;
 }
 
+std::vector<iDynTree::Matrix4x4> KinDynComputations::getWorldTransformsAsHomogeneous(const std::vector<std::string>& frameNames)
+{
+    std::vector<iDynTree::Matrix4x4> worldTransforms;
+    int numberOfTransforms=frameNames.size();
+    for(int number=0; number<numberOfTransforms; number++)
+    {
+        std::string frameName=frameNames[number];
+        int frameIndex = getFrameIndex(frameName);
+        if( frameIndex < 0 )
+        {
+            reportError("KinDynComputations","getWorldTransformsAsHomogeneous", "requested frameName not found in model");
+            worldTransforms.push_back(Transform::Identity().asHomogeneousTransform());
```

Comment:
Sorry, that will not be enough, because people do not read the output of the code. If there is an error, and error should be reported in someway to the called, for example returning an empty vector.

Reply:
Ok, I was following the logic implemented in other iDyntree Functions. Example : 
https://github.com/robotology/idyntree/blob/a92bdf9ca50ef5826034e190ba3f31f6e133e0ed/src/high-level/src/KinDynComputations.cpp#L892

Most of the getTransform functions have that logic, do you want me to change those as wel now that I'm at it?

Reply:
Now this will happen when a wrong frame is requeste ( only this function though ):
![image](https://user-images.githubusercontent.com/11043189/78924637-77b31a00-7a9a-11ea-8eec-53d0647aa345.png)